package/network/services/hostapd/files/hostapd.sh

```diff
@@ -376,6 +380,27 @@ hostapd_set_bss_options() {
 		append bss_conf "iapp_interface=$ifname" "$N"
 	}
 
+	json_get_vars ieee80211v
+	set_default ieee80211v 0
+	if [ "$ieee80211v" -eq "1" ]; then
+		json_get_vars time_advertisement time_zone wnm_sleep_mode bss_transition \
+			proxy_arp na_mcast_to_ucast
+
+		set_default time_advertisement 0
+		set_default wnm_sleep_mode 0
+		set_default bss_transition 0
+		set_default proxy_arp 0
+		set_default na_mcast_to_ucast 0
+
+		append bss_conf "time_advertisement=$time_advertisement" "$N"
+		[ -n "$time_zone" ] && append bss_conf "time_zone=$time_zone" "$N"
```

Comment:
Maybe we should get this from timezone in /etc/config/system instead of setting the same setting in different locations?

Reply:
hostapd.conf
```
# Local time zone as specified in 8.3 of IEEE Std 1003.1-2004:
# stdoffset[dst[offset][,start[/time],end[/time]]]
#time_zone=EST5
```
/etc/config/system
```
option timezone 'UTC'
```
Yes, the format looks like the same.
I think we can discuss better about this in the dedicated PR.

Reply:
I had already checked. Mine is ```option timezone 'EET-2EEST,M3.5.0/3,M10.5.0/4'``